diff --git a/node_modules/@nomiclabs/hardhat-etherscan/dist/src/network/prober.js b/node_modules/@nomiclabs/hardhat-etherscan/dist/src/network/prober.js
index cc55d54..206a6ce 100644
--- a/node_modules/@nomiclabs/hardhat-etherscan/dist/src/network/prober.js
+++ b/node_modules/@nomiclabs/hardhat-etherscan/dist/src/network/prober.js
@@ -24,6 +24,7 @@ var NetworkID;
     NetworkID[NetworkID["OPTIMISTIC_KOVAN"] = 69] = "OPTIMISTIC_KOVAN";
     // Polygon
     NetworkID[NetworkID["POLYGON"] = 137] = "POLYGON";
+    NetworkID[NetworkID["POLYGON_MUMBAI"] = 80001] = "POLYGON_MUMBAI";
 })(NetworkID || (NetworkID = {}));
 const networkIDtoEndpoints = {
     [NetworkID.MAINNET]: {
@@ -78,6 +79,10 @@ const networkIDtoEndpoints = {
         apiURL: "https://api.polygonscan.com/api",
         browserURL: "https://polygonscan.com",
     },
+    [NetworkID.POLYGON_MUMBAI]: {
+        apiURL: "https://api-testnet.polygonscan.com/api",
+        browserURL: "https://mumbai.polygonscan.com/"
+    },
 };
 async function getEtherscanEndpoints(provider, networkName) {
     if (networkName === plugins_1.HARDHAT_NETWORK_NAME) {
diff --git a/node_modules/@nomiclabs/hardhat-etherscan/src/network/prober.ts b/node_modules/@nomiclabs/hardhat-etherscan/src/network/prober.ts
index 4c5b302..62501a3 100644
--- a/node_modules/@nomiclabs/hardhat-etherscan/src/network/prober.ts
+++ b/node_modules/@nomiclabs/hardhat-etherscan/src/network/prober.ts
@@ -35,6 +35,7 @@ enum NetworkID {
   OPTIMISTIC_KOVAN = 69,
   // Polygon
   POLYGON = 137,
+  POLYGON_MUMBAI = 80001,
 }
 
 const networkIDtoEndpoints: NetworkMap = {
@@ -90,6 +91,10 @@ const networkIDtoEndpoints: NetworkMap = {
     apiURL: "https://api.polygonscan.com/api",
     browserURL: "https://polygonscan.com",
   },
+  [NetworkID.POLYGON_MUMBAI]: {
+    apiURL: "https://api-testnet.polygonscan.com/api",
+    browserURL: "https://mumbai.polygonscan.com/"
+  },
 };
 
 export async function getEtherscanEndpoints(
